packages/common/src/services/remote-config/defaults.ts

```diff
@@ -70,7 +70,8 @@ export const remoteConfigStringDefaults: {
   [StringKeys.STRIPE_ALLOWED_COUNTRIES]: '',
   [StringKeys.STRIPE_DENIED_REGIONS]: '',
   [StringKeys.COINBASE_PAY_ALLOWED_COUNTRIES_2_LETTER]: '',
-  [StringKeys.STRIPE_ALLOWED_COUNTRIES_2_LETTER]: ''
+  [StringKeys.STRIPE_ALLOWED_COUNTRIES_2_LETTER]: '',
+  [StringKeys.AUDIO_FEATURES_DEGRADED_TEXT]: null
```

Comment:
reminder that we need to delete the remote var to remove properly though empty string will work